test/hosting/hosting.tests.csproj

```diff
@@ -15,7 +15,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="xunit" Version="2.3.1" />
+    <PackageReference Include="xunit" Version="2.4.0" />
     <!-- DotNetCliToolReference element specifies the CLI tool that the user wants to restore in the context of the project. -->
     <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
```

Comment:
@bergmeister We skipped the old version :-)

Reply:
Did you mean `dotnet-xunit`? Version 2.4.0 of it is still in preview
https://www.nuget.org/packages/dotnet-xunit/2.4.0-beta.1.build3958

Reply:
Yes, we add it in csharp.tests.csproj in the PR.

Reply:
Ok, I see what you mean now. If it's not essential, then I'd still just wait for 2.4.0 of this package to become RTM.

Reply:
We can wait. My comment is informational only.